src/schema/schema_util.c

```diff
@@ -185,3 +185,57 @@ __wt_name_check(WT_SESSION_IMPL *session, const char *str, size_t len, bool chec
     __wt_scr_free(session, &tmp);
     return (ret);
 }
+
+/*
+ * __wt_schema_construct_table_config --
+ *     Construct the in-memory representation of a table's configuration.
+ */
+int
+__wt_schema_construct_table_config(WT_SESSION_IMPL *session, const char **config, WT_TABLE *table)
```

Comment:
Can the arg be named `cfg`?  We have a naming idiom that `config` is a simple (char *) string.  And an array of such strings with a NULL as terminator (generally const char ** or const char *cfg[]) is `cfg`.  It makes it easier to read.

Reply:
I have removed this function and ensured that i only used `cfg`.